components/lib/crash-sentry/src/main/java/mozilla/components/lib/crash/sentry/SentryService.kt

```diff
@@ -120,7 +120,13 @@ class SentryService(
     @VisibleForTesting
     internal fun initSentry() {
         SentryAndroid.init(applicationContext) { options ->
+            // Disable uncaught no native exceptions from being reported.
```

Comment:
I would point out here in the docs that we have our own uncaught exception handler so we don't need Sentry's default one.

Reply:
👍🏽